docs/notebooks/tutorial.ipynb

```diff
@@ -161,7 +167,7 @@
     }
    ],
    "source": [
-    "!run_algo.py -f ../../zipline/examples/buyapple.py --start 2000-1-1 --end 2014-1-1 -o buyapple_out.pickle"
+    "!zipline run -f ../../zipline/examples/buyapple.py --start 2000-1-1 --end 2014-1-1 -o buyapple_out.pickle"
```

Comment:
I guess since we don;t want the results in memory this is fine, you can pass `-f` and `-o` to the magic though